controllers/opentelemetrycollector_controller.go

```diff
@@ -83,31 +83,31 @@ func (r *OpenTelemetryCollectorReconciler) onPlatformChange() error {
 	}
 	r.muTasks.Unlock()
 
-	if err := r.addRouteTask(plt, routesIdx); err != nil {
+	if err := r.addRouteTask(openshiftRoutesAvailable, routesIdx); err != nil {
 		return err
 	}
 
-	return r.removeRouteTask(plt, routesIdx)
+	return r.removeRouteTask(openshiftRoutesAvailable, routesIdx)
 }
 
-func (r *OpenTelemetryCollectorReconciler) addRouteTask(plt platform.Platform, routesIdx int) error {
+func (r *OpenTelemetryCollectorReconciler) addRouteTask(openshiftRoutesAvailable openshift_routes.OpenShiftRoutesAvailability, routesIdx int) error {
```

Comment:
Can we use a shorter name?

Reply:
Which one? I'm open for the change but I don't know what name should I use

Reply:
```suggestion
func (r *OpenTelemetryCollectorReconciler) addRouteTask(ora openshift_routes.OpenShiftRoutesAvailability, routesIdx int) error {
```